content/en/docs/reference/setup-tools/kubeadm/kubelet-integration.md

```diff
@@ -0,0 +1,88 @@
+
+##### The dropin for the kubelet that consequently is needed for this to work
+
+```
+[Service]
+Environment="KUBELET_KUBECONFIG_ARGS=--bootstrap-kubeconfig=/etc/kubernetes/bootstrap-kubelet.conf --kubeconfig=/etc/kubernetes/kubelet.conf"
+Environment="KUBELET_CONFIG_ARGS=--config=/var/lib/kubelet/config.yaml"
+# This is a file that "kubeadm init" and "kubeadm join" generates at runtime, populating the KUBELET_KUBEADM_ARGS variable dynamically
+EnvironmentFile=-/var/lib/kubelet/kubeadm-flags.env
+# This is a file that the user can use for overrides of the kubelet args as a last resort. Preferably, the user should use
+# the .NodeRegistration.KubeletExtraArgs object in the configuration files instead. KUBELET_EXTRA_ARGS should be sourced from this file.
+EnvironmentFile=-/etc/default/kubelet
+ExecStart=
+ExecStart=/usr/bin/kubelet $KUBELET_KUBECONFIG_ARGS $KUBELET_CONFIG_ARGS $KUBELET_KUBEADM_ARGS $KUBELET_EXTRA_ARGS
+```
+
+This is the dropin file the `kubeadm` _deb/rpm package_ ships for the _kubelet_, in the path `/etc/systemd/system/kubelet.service.d/10-kubeadm.conf`. What to note here:
+
+- The KubeConfig file to use for the TLS Bootstrap `/etc/kubernetes/bootstrap-kubelet.conf` . Not used if `/etc/kubernetes/kubelet.conf` already exists.
+
+- The KubeConfig file with the unique kubelet identity `/etc/kubernetes/kubelet.conf`
+
+- The pointer to the file containing the kubelet's ComponentConfig: `/var/lib/kubelet/config.yaml`
+
+- The sourcing of the dynamic environment file `/var/lib/kubelet/kubeadm-flags.env` which should contain `KUBELET_KUBEADM_ARGS`
+
+- The sourcing of the `/etc/default/kubelet` (for debs), or `/etc/systconfig/kubelet` (for rpms) environment file that can contain the `KUBELET_EXTRA_ARGS` variable with flag overrides. As `KUBELET_EXTRA_ARGS` is last in the flag chain, it has the highest priority.
+
+  ### Things to keep in mind when packaging kubeadm, kubelet, etc. for a non-systemd system
+
+The deb/rpm packages shipped with the Kubernetes releases are the following:
+
+- `kubeadm` . Ships the `/usr/bin/kubeadm` CLI tool and the systemd dropin for the kubelet described above (in `/etc/systemd/system/kubelet.service.d/10-kubeadm.conf`).
```

Comment:
few spots have a space after a code word and before the period such as this line "`kubeadm` ."